gradle.properties

```diff
@@ -1,11 +1,11 @@
 # Project
-version=2.2
-version_patch=1
+version=2.3
+version_patch=0
 group=me.hypherionmc.sdlink
 library_ver=3.0.12
 
 # Common
-minecraft_version=1.19.3
+minecraft_version=1.19.4
```

Comment:
Same comment as before. Can't we use the same 1.19.3 code for both?

Reply:
It does work, updated this file aswell